### PR TITLE
Update active-directory-rights-manage-mobile-device.md

### PR DESCRIPTION
**Pull request by krishnacloud123**
Updated the document 

Removed the following text  and included https://docs.microsoft.com/en-us/azure/information-protection/known-issues#adrms-protected-files-on-android-devices 

```powershell
Add-AdfsClient -Name "Azure Information Protection app for Android" -ClientId "ECAD3080-3AE9-4782-B763-2DF1B1373B3A" -RedirectUri @("com.microsoft.rms-sharing-for-android://authorize")
```
```powershell
Grant-AdfsApplicationPermission -ClientRoleIdentifier 'ECAD3080-3AE9-4782-B763-2DF1B1373B3A' -ServerRoleIdentifier api.rms.rest.com -ScopeNames "openid"

### CODE REVIEW

File: Azure-RMSDocs/active-directory-rights-manage-mobile-device.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ You can download the Active Directory Rights Management Services (AD RMS) mobile
 
 For additional information, download the "Microsoft Azure Information Protection" white paper and accompanying scripts from the [Microsoft Download Center](https://www.microsoft.com/download/details.aspx?id=40333).
 
+Please note AIP mobile app on Android no longer support ADRMS protected files [Review Known issues ](https://docs.microsoft.com/en-us/azure/information-protection/known-issues#known-issues-for-the-aip-viewer)
```
Comment (ShannonLeavitt) [inline Azure-RMSDocs/active-directory-rights-manage-mobile-device.md:53]:
```suggestion
Please note AIP mobile app on Android no longer support ADRMS protected files. [Review known issues](https://docs.microsoft.com/azure/information-protection/known-issues#known-issues-for-the-aip-viewer).
```
Minor edits

**Comment by ShannonLeavitt** [comment]
Hi @batamig - When you review these changes, you might want to recommend an update to line 199. I've also added suggestions for line 53.

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by batamig** [comment]
Hi @krishnacloud123, can you please re-make this PR in the private repo so that you can sign off directly? Please contact me offline if you need more information about how to get there. You might want to consider the suggestions made here. 
We have some maintenance to do on this public repo but need to have these PRs closed first. Thanks!

**Comment by batamig** [comment]
Closing out old PRs
#please-close

**Pull request closed without merging**